src/redux/reducers/employer.ts

```diff
@@ -11,7 +11,7 @@ const initialState = {
   error: null,
 }
 
-const test = (state = initialState, action: employerActions) => {
+const test = (state = initialState, action: EmployerActions) => {
   switch (action.type) {
     case REGISTER_EMPLOYER_REQUEST:
       return { ...state, loading: true, credentials: '' }
```

Comment:
You should change this also. When `REGISTER_EMPLOYER_REQUEST` is dispatched, you'd want to store the credential payload in the store, so that you can let the saga extract the `credential` from the store and make the api request

Reply:
You mean store to local storage? I don't use persisted-reducer... ;) I will use thunk then to put the crdts to a save place. :))